try {
  dsfdf;
} catch (error) {
  console.log(error); //error.name   //error.message
} finally {
  //it will run irrespective of try and catch always
}
